python/lsst/ts/adamSensors/adamSensorsCSC.py

```diff
@@ -26,23 +34,98 @@ def __init__(self, config_dir=None, initial_state=salobj.State.STANDBY, initial_
     async def begin_start(self, id_data):
         await super().begin_start(id_data)
         self.adam = AdamModel(self.log, simulation_mode=True)
```

Comment:
You are passing `simulation_mode=True` here. Is this right? shouldn't you be checking whether the CSC is in simulation mode or not?